BootloaderCorePkg/Library/SmbiosInitLib/SmbiosTemplate.c

```diff
@@ -136,6 +136,42 @@ SMBIOS_TABLE_TYPE2    mBaseBoardInfo = {
   { 0 }                                         // ContainedObjectHandles
 };
 
+//
+// Default Chassis Info for SMBIOS Type 3
+//
+SMBIOS_TABLE_TYPE3 mDefaultChssisInfoInfo = {
```

Comment:
a typo: change mDefaultChssisInfoInfo to mDefaultChassisInfo 
I didn't see it is used in this patch. should it be used in SmbiosInit?

Reply:
i fix on the typo, and do the changes on mDefaultChassisInfo